### PR TITLE
Frontend test

### PR DESCRIPTION
**Pull request by IanChong1**
Completed all functionality needed for Sprint 2 backlog, each buttons should be working as it was stated in the Sprint Plans.

### CODE REVIEW

**Review by stxue1 — APPROVED** [review]
lgtm, only really needs documentation. A lot of this is also frontend stuff which I don't know too mcuh about, so someone else should review this

File: lib/common/database.dart [hunk 1]
```diff
@@ -226,7 +226,23 @@ class DatabaseService {
     }
     return Task();
   }
+  Future<List<Task>> getTasksOfName(String taskName) async {
```
Comment (anlhu) [inline lib/common/database.dart:229]: Can we add a newline here? A lot of files are not up to standard with the new coding standards, maybe we can fix them all at once then instead?

File: lib/common/database.dart [hunk 2]
```diff
@@ -226,7 +226,23 @@ class DatabaseService {
...
+    final allTasks = await users.doc(uid).collection("tasks")
+            .where("task name",  isEqualTo: taskName).get();
+    
+    if (allTasks.size == 0) {
+      return [];
+    }
```
Comment (anlhu) [inline lib/common/database.dart:235]: I think this can be deleted?

File: lib/main.dart [hunk 3]
```diff
@@ -32,7 +41,7 @@ void main() async {
   // test_tasks();
   //Uncomment below line and comment out above line to see weekView + dayView UI
 
-  runApp(GetMaterialApp(home: weekView()));
+  //runApp(GetMaterialApp(home: weekView()));
```
Comment (anlhu) [inline lib/main.dart:45]: was this intentional?

**Review by anlhu — APPROVED** [review]
Couldnt understand the frontend much but looks good.
I'll make a note to come back to some of these

**Pull request merged**